sys/linux/filesystem.txt.const

```diff
@@ -56,7 +56,7 @@ MS_SYNCHRONOUS = 16
 MS_UNBINDABLE = 131072
 OPEN_TREE_CLOEXEC = 524288
 OPEN_TREE_CLONE = 1
-SYZ_MOUNT_IMAGE_TIMEOUT = 50
+SYZ_MOUNT_IMAGE_TIMEOUT = 5000
```

Comment:
What's the typical execution time of mount calls?
5 seconds is the total program execution timeout. It's very high. I assume some mounts can actually hang forever (fuse, ntfs, 9p with virtio transport, etc). That may the reason for lower number of test executions in the benchmarks. Also for all of these filesystems that require "server replies" to be mounted (fuse/ntfs/9p) we _need_ to detect it as hanged and proceed with execution of subsequent syscalls that supply replies and will eventually make the mount unblock and finish. If the timeout is set to the total program timeout, it will never happen (we will never provide replies and these mounts will never succeed).